back/routes/review.js

```diff
@@ -75,7 +75,7 @@ async function activeFeedbackQuery(objectId){
   if (experienceInfo) {
     return experienceInfo.toJSON().activeFeedback
   }
-  const allFeedbackQuery = new Parse.Query('Feedback');
+  const allFeedbackQuery = new Parse.Query('Preference').equalTo('forExperience', true);
```

Comment:
Oh, that is because there are some values like the distance, where the experienceMaker can not ask for feedback.
Because that is like a fixed value.